tests/cases/fourslash/getOutliningSpansForRegions.ts

```diff
@@ -0,0 +1,46 @@
+/// <reference path="fourslash.ts"/>
+
+////// basic region
+////[|// #region
+////
+////// #endregion|]
+////
+////// region with label
+////[|// #region label1
+////
+////// #endregion|]
+////
+////// region with extra whitespace in all valid locations
+////[|             //              #region          label2    label3
+////
+////        //        #endregion|]
+////
+////// No space before directive
+////[|//#region label4
+////
+//////#endregion|]
+////
+////// Nested regions
+////[|// #region outer

```

Comment:
As far as I can tell, fourslash doesn't currently gather that information.
I can try to add the functionality if you would like me to though.